src/models/blockchain/midnight/block.rs

```diff
@@ -0,0 +1,142 @@
+//! Midnight block data structures.
+//!
+//! Note: These structures are based on the Midnight RPC implementation:
+//! <https://github.com/midnightntwrk/midnight-node/blob/39dbdf54afc5f0be7e7913b387637ac52d0c50f2/pallets/midnight/rpc/src/lib.rs>
+
+use serde::{Deserialize, Serialize};
+use std::ops::Deref;
+
+use crate::models::MidnightRpcTransactionEnum;
+
+/// Represents a Midnight block
+///
+/// <https://github.com/midnightntwrk/midnight-node/blob/39dbdf54afc5f0be7e7913b387637ac52d0c50f2/pallets/midnight/rpc/src/lib.rs#L214-L218>
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct RpcBlock {
+	#[serde(rename = "header")]
```

Comment:
Do we need renames given they are single words?

Reply:
I guess it was just for consistency. We need it for `transactions_index`, but can remove for header and body

Reply:
Pushed in #218